src/module.ts

```diff
@@ -212,5 +212,12 @@ export default defineNuxtModule<ModuleOptions>({
     // Composables
 
     addImportsDir(resolve(runtimeDir, 'composables'))
+
+    nuxt.options.alias['#ui'] = addTemplate({
```

Comment:
What's this for? Can you not import the `Form` type  from `@nuxthq/ui/runtime/types`?

Reply:
I initially tried doing so `import { Form } from '@nuxthq/ui/dist/runtime/types'`, but somehow typescript can't resolve the import properly (I might have missed something here).

I found this through the [Module Author Guide](https://nuxt.com/docs/guide/going-further/modules#adding-templatesvirtual-files) and modules like @sidebase/nuxt-auth.

Reply:
Just pushed to fix this